tests/backup/backup_restore_basic_test.go

```diff
@@ -2763,3 +2764,458 @@ var _ = Describe("{BackupRestoreOnDifferentK8sVersions}", func() {
+var _ = Describe("{BackupCRs.MultipleRestoresOnHigherK8sVersion}", func() {
+
+	var (
+		backupNames          []string             // backups in px-backup
+		restoreNames         []string             // restores in px-backup
+		restoreLaterNames    []string             // restore-laters in px-backup
+		scheduledAppContexts []*scheduler.Context // Each Context is for one Namespace which corresponds to one App
+		preRuleNameList      []string
+		postRuleNameList     []string
+		scheduledClusterUid  string
+		cloudCredName        string
+		cloudCredUID         string
+		backupLocationUID    string
+		backupLocationName   string
+	)
+
+	var (
+		appList           = Inst().AppList
+		namespaceMapping  = make(map[string]string)
+		backupLocationMap = make(map[string]string)
+		labelSelectors    = make(map[string]string)
+		providers         = getProviders()
+	)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("BackupCRs.MultipleRestoresOnHigherK8sVersion", "deploy CRs (CRD + webhook); then backup; create two simulatanous restores on cluster with higher K8s version; one restore is Success and other PartialSuccess", nil, 83716)
+	})
+
+	It("Deploy CRs (CRD + webhook); Backup; two simulatanous Restores with one Success and other PartialSuccess. (Backup and Restore on different K8s version)", func() {
+
+		defer func() {
+			log.InfoD("switching to default context")
+			err1 := SetClusterContext("")
+			log.FailOnError(err1, "failed to SetClusterContext to default cluster")
+		}()
+
+		Step("Verify if app used to execute test is a valid/allowed spec (apps) for *this* test", func() {
+			log.InfoD("specs (apps) allowed in execution of test: %v", appsWithCRDsAndWebhooks)
+			for i := 0; i < len(appList); i++ {
+				contains := Contains(appsWithCRDsAndWebhooks, appList[i])
+				dash.VerifyFatal(contains, true, fmt.Sprintf("app [%s] allowed in execution of this test", appList[i]))
+			}
+		})
+
+		Step("verify kubernetes version of source and destination cluster", func() {
+			var srcVer, destVer semver.Version
+			log.InfoD("begin verification kubernetes version of source and destination cluster")
+
+			Step("Registering cluster for backup", func() {
+				log.InfoD("Registering cluster for backup")
+				ctx, err := backup.GetAdminCtxFromSecret()
+				log.FailOnError(err, "Fetching px-central-admin ctx")
+
+				err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+				dash.VerifyFatal(err, nil, "Creating source and destination cluster")
+
+				clusterStatus, err := Inst().Backup.GetClusterStatus(orgID, SourceClusterName, ctx)
+				log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", SourceClusterName))
+				dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", SourceClusterName))
+
+				scheduledClusterUid, err = Inst().Backup.GetClusterUID(ctx, orgID, SourceClusterName)
```

Comment:
How about srcClusterUid or sourceClusterUid instead of scheduledClusterUid?

Reply:
I used that name as it makes much more sense when you're doing deeper validation. While deeper validation code has not been added here yet, it will be later on. The depper validation expects clusters called "scheduledClusters" to clearly indicate where things have been scheduled, rather than assume that it's on something names "source" or "destination". This way, things become more clearly indicated.

You will also notice that `scheduledClusterUid` is used only for the purpose of knowing where things are scheduled as opposed to necessarily referring to the source. (which means, if anyone wants to, they can change it to point to the destination cluster).

Reply:
What will you do for a test case where we are scheduling on both source and destination?
Please rename this back to source cluster.